translations.js

```diff
@@ -47,11 +47,22 @@ var createDoc = function(attachment) {
 
```

Comment:
This is a pretty small file, and we've been changing it quite a bit recently. Time to ES6-a-fy?